mmdet/datasets/transforms/transforms.py

```diff
@@ -2878,3 +2878,148 @@ def __repr__(self):
         repr_str += f'mask_occluded_thr={self.mask_occluded_thr}, '
         repr_str += f'selected={self.selected})'
         return repr_str
+
+
+@TRANSFORMS.register_module()
```

Comment:
Suggest create a new PR to add RandomErasing with its UT so that it could be merged quickly.